.github/pull_request_template.md

```diff
@@ -37,12 +37,12 @@ I have verified these changes via:
 
 - [ ] Code inspection only, or
 - [ ] Newly added/modified unit tests, or
-- [ ] No unit tests but ran on a real repository, or
+- [ ] No unit tests, but ran on a real repository, or
 - [ ] Both unit tests + ran on a real repository
 
 The public repository: <URL>
 
-<!-- Do you have any suggestions about this PR template? Edit it here: https://github.com/renovatebot/renovate/edit/main/.github/pull_request_template.md -->
+<!-- If you have any suggestions about this PR template, edit it here: https://github.com/renovatebot/renovate/edit/main/.github/pull_request_template.md -->
 
 <!-- Please do not force push to your PR's branch after you have created your PR, as doing so forces us to review the whole PR again. This makes it harder for us to review your work because we don't know what has changed. -->
-<!-- PRs will always be squashed by us when we merge your work. Commit as many times as you need in this branch. -->
+<!-- PRs will always be squashed by us when we merge your work. You can commit as many times as you need in this branch. -->
```

Comment:
I like the old one better.

Reply:
I think the new one sounds a bit more permissive and tells a contributor not to worry if they need a few more commits than originally intended. I also think it sounds a bit more welcoming.

If you really don't agree, I can change it back, but I personally think the suggested change does come across a little bit better.